app/presenters/tasks/other_info.rb

```diff
@@ -6,5 +6,9 @@ class OtherInfo < Generic
     def path
       edit_steps_other_info_path(application)
     end
+
+    def completed?
+      Steps::OtherInfoForm.build(application).valid?
```

Comment:
you get this for free by setting the `FORM` value

Reply:
I believe my build broke and this sorted it. Will remove and see how i get on